applications/DEMApplication/custom_utilities/inlet.cpp

```diff
@@ -42,8 +42,12 @@ namespace Kratos {
 
     /// Constructor
 
-    DEM_Inlet::DEM_Inlet(ModelPart& inlet_modelpart, const int seed): mInletModelPart(inlet_modelpart)
-    {
+    DEM_Inlet::DEM_Inlet(ModelPart& inlet_modelpart, const int seed):
+                        DEM_Inlet(inlet_modelpart, Parameters(R"({"dem_inlets_settings" : {}})"), seed){}
```

Comment:
```suggestion
                        DEM_Inlet(inlet_modelpart, Parameters(R"({})"), seed){}
```
I think "dem_inlets_settings" can stay outside

Reply:
But it is required in the other constructor to have these settings...The initial idea (as explained in the description) was to pass the whole parameters object to the inlet. However, I agree to only pass the inlet settings for now. We can postpone this discussion until the inlets_manager exists.